Sources/Components/SelectionBoxes/AnimatedCheckboxView.swift

```diff
@@ -30,10 +30,18 @@ public class AnimatedCheckboxView: AnimatedSelectionView {
     }
 
     private func setImages() {
-        let framesPerSecond = AnimatedSelectionView.framesPerSecond
-        guard let selectedImages = UIImage.animatedImageNamed("checkbox-selected-", duration: 20 / framesPerSecond)?.images,
-            let unselectedImages = UIImage.animatedImageNamed("checkbox-unselected-", duration: 14 / framesPerSecond)?.images else {
-                fatalError("Could not instantiate animation images")
+        var selectedImages = [UIImage]()
+        for index in 0 ..< 20 {
+            if let image = UIImage(named: "checkbox-selected-\(index)", in: FinniversKit.bundle, compatibleWith: nil) {
```

Comment:
It would maybe be nicer to have these defined in a `[FinniversImageAsset]`, so we can still use our rename-proof way of accessing resource images

Reply:
I don't think we can since that's autogenerated. We should still maybe extract into a variable.

Reply:
Ah, wasn't aware these images were not part of the assets catalog that generates `FinniversImageAsset`